torch_glow/src/CachingGraphRunner.h

```diff
@@ -20,18 +20,52 @@
 #include <torch/csrc/jit/custom_operator.h>
 #include <torch/csrc/jit/ir.h>
 
-#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Runtime/HostManager/HostManager.h"
 
 namespace glow {
 
 /// Responsible for maintaining a mapping from PyTorch subgraphs and their
-/// unique input types to compiled Glow Functions.
+/// unique input types to a compiled Glow Function.
 class CachingGraphRunner {
-  /// Glow ExecutionEngine.
-  glow::ExecutionEngine executionEngine_;
+  struct PerGlowGraphInfo {
+    std::vector<glow::Placeholder *> inputPlaceholders;
+    std::vector<glow::Placeholder *> outputPlaceholders;
+
+    /// Name of the Glow function maintained by HostManager for this subgraph.
+    std::string functionName;
+
+    /// The unique hash for the subgraph and input shapes represented by this
+    /// PerGlowGraphInfo.
+    size_t hash;
```

Comment:
it's a key in the map, why to store it again?

Reply:
To double check it against the argument spec in runGraphImpl, maybe it's not worth it